src/commands/__tests__/cycle.test.js

```diff
@@ -11,10 +11,14 @@ describe(testContext(__filename), function () {
       this.notify = msg => {
         this.notifications.push(msg)
       }
+      this.formatError = msg => {
+        this.errors.push(msg)
+      }
```

Comment:
Not a big deal, but I'm not wild about hooking into `formatError` here instead of `notify` in these tests. The way they are now we could format the error and NOT send it to the user and the tests would still pass.